### PR TITLE
Replaced logos and images with new Peace Corps branding

### LINKED ISSUE #79: Replacing the Peace Corps logo with the new redesigned logo
**Description**
>As a user, abiding the Peace Corps branding guidelines is important.
>I would want the new logo to be displayed instead,
>so that the guidelines are being followed.

**Acceptance Criteria**
We currently have various new logos, which will be used depending upon the screen size. Hence a responsive design for the page needs to be implemented. New logos should be displayed instead of the old one.

**Definition of Done**
Sending a PR and getting it reviewed by mentor(s) .

**Mocks**
Adding the new logo here.

screen shot 2018-02-19 at 12 18 16 am

These are the new logos
screen shot 2018-02-25 at 7 27 51 pm

**Estimation**
5-6 hours

### PR DESCRIPTION
**Pull request by zyang7**
### Description
Replaced all instances of the old Peace Corps logo with the new logos, including images with the logo edited into it.
Fixes #79 

### Type of Change:
- User Interface

### How Has This Been Tested?
- Check logo used on localhost:3000 page
screen shot 2018-04-07 at 1 49 25 pm
- Check logo used on Introduction module
screen shot 2018-04-07 at 1 48 57 pm

### Checklist:
- [x ] I have performed a self-review of my own code or materials

### CODE REVIEW

File: config/settings.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ module.exports = {
     basePath: 'http://localhost:3000/',
     googleAuth: {
         clientID: '701417346516-33qb38qk06480vh9vg91btfletf6nlqr.apps.googleusercontent.com',
-        clientSecret: '45rNBMN0HsoYJmsyxZ3qkHeC',
+        clientSecret: 'bf538d9651594b4b99667f287cdc9fd5',
```
Comment (shreyans29) [inline config/settings.js]: Is this your secret key? If not, is there any reason behind changing it.

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "core-js": "^2.4.1",
```
Comment (shreyans29) [inline package.json]: This pull request is for changing the Peace Corps logo. If we need to have flexibility in versions we need to create a new issue for it and then proceed.

**Comment by Raxerz** [comment]
@Buddhiprabha Can these peace corps logo be approved?

**Comment by Buddhiprabha** [comment]
@Raxerz yes, but this PR does too many things. Either the all other changes should be removed or this PR should be closed. @zyang7 please address the comments.

**Comment by Raxerz** [comment]
@Buddhiprabha the comments on this PR have not been addressed yet, it can be closed.

**Comment by Buddhiprabha** [comment]
Closing due to inactivity.

**Pull request closed without merging**